homeassistant/components/switch/zha.py

```diff
@@ -41,9 +61,18 @@ def async_turn_on(self, **kwargs):
         """Turn the entity on."""
         yield from self._endpoint.on_off.on()
         self._state = 1
+        self.schedule_update_ha_state()
 
     @asyncio.coroutine
     def async_turn_off(self, **kwargs):
         """Turn the entity off."""
         yield from self._endpoint.on_off.off()
         self._state = 0
+        self.schedule_update_ha_state()
```

Comment:
If you really need this use `async_schedule_update_ha_state()`. But since this is a non polling entity, doesn't the device feedback the state change to the entity via `async_update`? Then this isn't needed.